synchrad/calc.py

```diff
@@ -514,7 +515,7 @@ def _init_comm(self):
         ctx_kw_args = {}
         if self.Args['ctx'] is None:
             ctx_kw_args['interactive'] = True
-        elif self.Args['ctx'] is 'mpi':
+        elif self.Args['ctx'] == 'mpi':
             # temporal definition, assumes default 0th platform
             ctx_kw_args['answers'] = [0, self.rank]
         elif self.Args['ctx'] is not False:
```

Comment:
with suggested code this part would be unused, and there will be no fall-back to interactive choice of the device. We'll need to keep this option and avoid mandatory automatization of the context choice.